src/app/analytics/analytics.factory.js

```diff
@@ -150,7 +191,12 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
       try {
         // Build products variable
         this.buildProductVar(cartData)
-
+        // Send GTM Advance Ecommerce event
+        if (typeof $window.dataLayer !== 'undefined') {
+          $window.dataLayer.push({
+            event: callType === 'mini-cart' ? 'view-mini-cart' : 'view-cart'
+          })
+        }
```

Comment:
So for this, I noticed ```callType``` wasn't actually being used so I decided to use it. They want to differentiate between viewing the mini cart and then actually going to view the cart. So where I call the analytics function for the view-mini-cart event, I pass 'mini-cart' as a parameter so we can differentiate the two events. Let me know if I should update the ```callType``` parameter name to something else.

Reply:
Since the current code doesn't use it, you could make it a boolean if you wanted

Reply:
Yeah, that sounds good. I'll change it to ```isMiniCart``` and then have a default value of false.